src/ceph-volume/plugin/zfs/tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py27, py34, py35, py36, flake8
+
+[travis]
+python =
+    3.6: py36
+    3.5: py35
+    3.4: py34
+    2.7: py27
+
+[testenv:flake8]
+basepython = python
+deps = flake8
+commands = flake8 ceph_volume_zfs
+
+[testenv]
+setenv =
+    PYTHONPATH = {toxinidir}
+
+commands = python setup.py test
```

Comment:
how does this tool test itself?

Reply:
I have not given that any thought yet, being a relative newbie in python.